src/time.cpp

```diff
@@ -37,4 +37,24 @@ namespace libtorrent { namespace aux {
 	time_point time_now() { return clock_type::now(); }
 	time_point32 time_now32() { return time_point_cast<seconds32>(clock_type::now()); }
 
+	time_t to_time_t(const time_point32 &tp)
+	{
```

Comment:
I think it might be nice to add:
```
using std::chrono::system_clock;
```
here, to make the names a bit shorter in this function. Also in the other function, `from_time_t`.